app/forms/collection_edit_form.rb

```diff
@@ -21,6 +27,8 @@ def self.model_attributes(attrs)
     attrs.dup.each do |key, value|
       if value.is_a? Array
         attrs[key].map! { |v| to_hash(v) }
+      elsif value.nil? || value.to_s.strip.blank?
```

Comment:
@mcritchlow Yes, just for sanitize/delete empty attributes but it looks more like a upstream issue that should do the work, while I am seeing empty triples inserted which is not a big issue. How about I removing that conditional check for now?